docs: prometheus integration (#2286)
glasskube/distr: website/src/content/docs/docs/self-hosting/prometheus.md

| `distr_deployment_status_timestamp_seconds` | Gauge | `organization`, `customerorganization`, `deploymenttarget`, `deploymentid`, `application`, `applicationversion`           | Unix timestamp of the most recent status update for a deployment                                        |

The `distr_deployment_status` metric uses a one-hot encoding pattern: for each deployment, there is one time series per possible status value (`healthy`, `running`, `progressing`, `error`), where exactly one is set to `1` and the rest to `0`.

### Go runtime and process metrics

In addition to the Distr-specific metrics, the endpoint exposes standard Go runtime and OS process metrics:

- **Go runtime** (`go_*`): goroutine count, GC duration, memory allocation, heap usage, GC configuration
- **Process** (`process_*`): CPU time, memory usage (resident and virtual), open file descriptors, network I/O, process start time

## Use cases

### Custom alerting for customer deployments

Distr includes built-in [alerts](/docs/agents/alerts/) that notify users by email when deployments become unhealthy or resource usage exceeds thresholds.

> For more advanced alerting workflows, the Prometheus metrics endpoint can be combined with [Alertmanager](https://github.com/prometheus/alertmanager) or [Grafana Alerts](https://grafana.com/docs/grafana/latest/alerting/) to build custom error reporting on top of the integrated Distr alerts.

Reviewer: ```suggestion
For more advanced alerting workflows, the Prometheus metrics endpoint can be combined with [Alertmanager](https://github.com/prometheus/alertmanager) or [Grafana Alerts](https://grafana.com/docs/grafana/latest/alerting/) to build custom error reporting independently from the integrated Distr alerts.
```
